### PR TITLE
chore(rdws): fix enable platform field for RDWS

### PR DESCRIPTION
**Pull request by huanjani**
Fix broken platform functionality for request-driven web services.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by efekarakus — APPROVED** [review]

File: templates/workloads/services/rd-web/manifest.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ image:
 cpu: {{.InstanceConfig.CPU}}
 # Amount of memory in MiB used by the task.
 memory: {{.InstanceConfig.Memory}}
-{{- if .Platform}}
+{{- if .InstanceConfig.Platform}}
 platform: {{.Platform}}   # See https://aws.github.io/copilot-cli/docs/manifest/rd-web-service/#platform
```
Comment (Lou1415926) [inline templates/workloads/services/rd-web/manifest.yml]: Does this need to be 
```
platform: {{ .InstanceConfig.Platform}}
```
?

**Review by Lou1415926 — COMMENTED** [review]
Looks good! Just one small comment to double check -

File: templates/workloads/services/rd-web/manifest.yml (hunk 1 above)
Comment (huanjani) [reply templates/workloads/services/rd-web/manifest.yml]: gah! yes, thank you. fixed.

**Review by Lou1415926 — APPROVED** [review]
:shipit:

**Pull request merged**